drivers/adc/adc_nrfx_saadc.c

```diff
@@ -96,8 +97,6 @@ static struct driver_data m_data = {
 	ADC_CONTEXT_INIT_SYNC(m_data, ctx),
 #if defined(ADC_BUFFER_IN_RAM)
 	.samples_buffer = adc_samples_buffer,
-	.user_buffer = NULL,
-	.active_channels = 0,
```

Comment:
This change should be actually done in a separate commit as it is not related to negative sample values.